src/strava_py/process_data.py

```diff
@@ -1,7 +1,9 @@
+import glob
+
 import fit2gpx
 import gpxpy
-import os.path
 import pandas as pd
+from rich.progress import track
```

Comment:
It's common to sort imports into groups: stdlib first, third-party second, local last.

I used this tool: https://pycqa.github.io/isort/

```sh
python -m pip install -U isort
isort [path or filename]
```